torch/utils/cpp_extension.py

```diff
@@ -359,6 +359,14 @@ def unix_cuda_flags(cflags):
                     ['--compiler-options', "'-fPIC'"] +
                     cflags + _get_cuda_arch_flags(cflags))
 
+        def convert_to_absolute_paths_inplace(paths):
+            # Helper function. See Note [Absolute include_dirs]
+            if not paths:
```

Comment:
nit: It's better as `if paths is not None` to make it clear that we're testing for non-None-ness, otherwise this might look like a check that paths is an empty list (which is unnecessary)